tools/cell_census_builder/__main__.py

```diff
@@ -5,15 +5,16 @@
 import os.path
 import sys
 from datetime import datetime, timezone
-from typing import List
+from typing import List, Union
```

Comment:
Union is also unused - should not be passing lint!

Reply:
fixed